styles/style.css

```diff
@@ -0,0 +1,571 @@
+  .product__item:nth-child(2) { display: grid; }
+
+  .product__items { justify-content: space-around; }
+  
+  .product__items { padding: 0 calc(6.25em - 9.77vw); }
+}
+
+@media only screen and (min-width: 768px) {
+  .header__title { font-size: calc(9.16em - 7.81vw); }
+
+  .header__link,
+  .story__link { font-size: calc(3.666em - 3.125vw); }
+
+  .product__header,
+  .info__header,
+  .stories__header { font-size: calc(4.5em - 3.125vw); }
+
+  .product__name { font-size: calc(3.66em - 3.12vw); }
+
+  .product__price { font-size: calc(4.83em - 4.3vw); }
+
+  .body > * { padding: 0 calc(11rem - 17.1875vw); }
+
+  .body > :not(.footer) { margin: 0 calc(18.36vw - 8.8125rem); }
+
+  .header {
+    /* padding: 0 4.5em; */
+    padding: 0 calc(1.16em + 3.9vw);
+    height: calc(56.41rem - 63.46vw);
+
+    /* grid-template-rows: 6.3em 8em; */
+    grid-template-rows: calc(21.1em - 17.34vw) calc(40em - 37.5vw);
+
+    /* background: url(../logos/rob-potter-422534-unsplash.png) no-repeat -10.5em/cover; */
+    background: url(../logos/rob-potter-422534-unsplash.png) no-repeat calc(49.23vw - 42em)/cover;
+  }
+
+  .product {
+    /* height: 34.8em; */
+    height: calc(62em - 31.87vw);
+
+    /* grid-template-rows: 5.26em min-content auto;
+    grid-template-rows: calc(4.62em + .75vw) 4em; */
+    grid-template-rows: calc(4.62em + .75vw) calc(10em - 7.03vw);
+  }
+
+  .product__item {
+    margin-left: 0;
+
+    /* height: 18.8em;
+    width: 15.5em; */
+    height: calc(44.92em - 30.61vw);
+    width: calc(38.74em - 27.23vw);
+  }
+
+  .product__link {
+    /* grid-template-rows: 16.2em 1.5em; */
+    grid-template-rows: calc(33em - 19.69vw) calc(7.5em - 7.03vw);
+  }
+
+  .product__header {
+    padding-right: calc(2em - 5.86vw);
+  }
+
+  .product__img {
+    /* width: 14.5em;
+    height: 13.5em; */
+    width: calc(34.5em - 23.44vw);
+    height: calc(31.5em - 21.09vw);
+  }
+
+  .product__clrs {
+    /* width: 4.1em; */
+    width: calc(19.3em - 17.81vw);
+  }
+
+  .product__clr {
+    width: calc(4.92em - 5.08vw);
+    height: calc(4.92em - 5.08vw);
+  }
+}
+
+@media only screen and (min-width: 868px) {
+  .product__item:nth-child(3) { display: grid; }
+}
+
+@media only screen and (min-width: 1024px) {
+  .header__title { font-size: 2.5em; }
+
+  .header__link,
+  .story__link { font-size: 1em; }
+
+
+  .product__header,
+  .info__header,
+  .stories__header { font-size: 1.83em; }
+
+  .product__name { font-size: 1em; }
+
+  .product__price { font-size: 1.16em; }
+
+  .body > :not(.footer) { margin: 0 calc(15.753vw - 7.144rem); }
+
+  .nav__hamb_none,
+  .nav__home_mob_none { display: none; }
+
+  .nav__home_desk { display: initial; }
+
+  .nav__link_col,
+  .nav__signin_col,
+  .nav__cart_col,
+  .nav__lang_col { height: unset; }
+
+  .nav__checkbox:checked ~ .nav__items { display: flex; }
+
+  .nav {
+    display: flex;
+    justify-content: space-between;
+    height: calc(14.05rem - 9.3vw);
+    min-height: 9.66em;
+  }
+  
+  .nav__items {
```

Comment:
Codacy found an issue: [Expected selector ".nav__items" to come before selector ".nav__checkbox:checked ~ .nav__items" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/r4/pullRequest?prid=5887702)